plugins/base/src/main/kotlin/DokkaBaseConfiguration.kt

```diff
@@ -8,12 +8,14 @@ data class DokkaBaseConfiguration(
     var customStyleSheets: List<File> = defaultCustomStyleSheets,
     var customAssets: List<File> = defaultCustomAssets,
     var separateInheritedMembers: Boolean = separateInheritedMembersDefault,
-    var footerMessage: String = defaultFooterMessage
+    var footerMessage: String = defaultFooterMessage,
+    var mergeNoExpectActualDeclarations: Boolean = mergeNoExpectActualDeclarationsDefault
```

Comment:
Would be nice to have a comment here with an example of what it's used for.

Also, what do you think about renaming it to `mergeImplicitExpectActualDeclarations`? (implicit as in `suggested though not directly expressed`, as opposed to `explicit`)